### PR TITLE
Fixes #3276 - PHP 7 multipage issue

### LINKED ISSUE #3276: PHP 7 multipage issue
Today I was debugging stuff on my forum and I found an issue with multipage. The issue happens in some pages that uses multipage. The issue happens if you specify a big number for page because in many pages there aren't conditions that check if specified page is greater then max page.
This is an example:
https://community.mybb.com/memberlist.php?page=999999999999999999
Many admin pages and some forum page are affected by this. It is a low bug but it should be fixed.

### PR DESCRIPTION
**Pull request by chack1172**

### CODE REVIEW

**Comment by euantorano** [comment]
This looks good. I'm leaving it open for now until 1.8.16 is released then we can merge.

**Comment by SvePu** [comment]
I've closed my PR #3277 ... This is the better fix.

**Comment by euantorano** [comment]
As 1.8.17 is a hot fix release, this will be part of 1.8.18. There isn't a milestone for 1.8.18 yet, but once there is I will assign it.

File: misc.php [hunk 1]
```diff
@@ -293,9 +302,6 @@
 	}
 
 	// Do Multi Pages
-	$query = $db->simple_select("helpdocs", "COUNT(*) AS total", "hid IN(".$db->escape_string($search['querycache']).")");
-	$helpcount = $db->fetch_array($query);
-
 	if($upper > $helpcount)
```
Comment (Shade-) [inline misc.php]: This doesn't make any sense; `$upper` is a number, `$helpcount` is an array.

File: private.php [hunk 2]
```diff
@@ -303,9 +312,6 @@
 	}
 
 	// Do Multi Pages
-	$query = $db->simple_select("privatemessages", "COUNT(*) AS total", "pmid IN(".$db->escape_string($search['querycache']).")");
-	$pmscount = $db->fetch_array($query);
-
 	if($upper > $pmscount)
```
Comment (Shade-) [inline private.php]: Likewise.

**Pull request merged**